mitosheet/src/mito/components/SearchBar.tsx

```diff
@@ -188,57 +193,95 @@ export const SearchBar: React.FC<SearchBarProps> = (props) => {
         setTotalMatches(e.target.value === '' ? 0 : undefined);
     }
 
+    const handleReplace = () => {
+        void mitoAPI.editReplace(uiState.selectedSheetIndex, searchValue ?? '', replaceValue ?? '');
+        getMatches();
```

Comment:
I think you might have to make handleReplace async, and then change the void -> async. Otherwise, you might have race conditions!